build/root/WORKSPACE

```diff
@@ -70,10 +70,10 @@ docker_pull(
 
 docker_pull(
     name = "debian-hyperkube-base-amd64",
-    digest = "sha256:f3a37c4d8700a5ff454d94a2bef7d165d287759cea737a621c20e4aa3891dbbb",
+    digest = "sha256:UPDATE",
```

Comment:
Will you update this when you push the image? Or is the manual sha no longer needed?

Reply:
yes, I left this here to intentionally break the build, but will fill in once pushed.